### ISSUE TITLE
Warning “A non-numeric value encountered” when using parentheses with LIMIT statement

### ISSUE THREAD

**Issue by JoBerkner** [phpmyadmin/phpmyadmin#15347]
### Describe the bug

A PHP warning occurs when running a valid SQL query. The warning message is:

`A non-numeric value encountered`

### To Reproduce

1. Go to phpmyadmin
2. Create two tables that share one column
3. Run the following query:
```sql
(SELECT name FROM tableA LIMIT 1)
UNION
(SELECT name FROM tableB LIMIT 1)
```
4. The warning occurs, but the results are also shown correctly.

### Expected behavior

The query should result without any warnings.

### Server configuration

- Operating system: (no info)
- Web server: (no info)
- Database version: MySQL 5.6.42
- PHP version: 7.2
- phpMyAdmin version: 4.9.0.1

### Client configuration

- Browser: Chrome
- Operating system: Windows

### Additional context

 This can also be reproduced with a query as basic as this:
`(SELECT * FROM table LIMIT 1)`
Basically always parentheses and `LIMIT` is used together.

**Comment by williamdes** [comment]
@JoBerkner can you help reproduce this issue ?
```sql
(SELECT name FROM event LIMIT 1) UNION (SELECT name FROM pet LIMIT 1)
```
https://demo.phpmyadmin.net/QA_4_9/db_sql.php?db=menagerie

**Comment by JoBerkner** [comment]
On the demo server you linked to I could reproduce the warning using this query:

`(SELECT * FROM information_schema.SCHEMATA LIMIT 1)`

**Comment by williamdes** [comment]
Reproduced using :
```sql
(SELECT "e" FROM dual LIMIT 1)
UNION
(SELECT "ee" FROM dual LIMIT 1)
```

**Comment by williamdes** [comment]
You need to edit inline the query and submit it again.

**Comment by williamdes** [comment]
I opened an issue on our parser https://github.com/phpmyadmin/sql-parser/issues/275

**Comment by shucon** [comment]
Hey, @williamdes should this issue be closed now?

**Comment by williamdes** [comment]
Sure !
It was fixed in https://github.com/phpmyadmin/sql-parser/releases/tag/4.5.0 and https://github.com/phpmyadmin/sql-parser/releases/tag/5.2.0

**Comment by williamdes** [comment]
Included in https://github.com/phpmyadmin/phpmyadmin/releases/tag/RELEASE_4_9_4 and https://github.com/phpmyadmin/phpmyadmin/releases/tag/RELEASE_5_0_1

Proof for 4.9: https://github.com/code-lts/phpmyadmin-releases/commit/02319230d31daa160655a996543e64170d89996f#diff-134cdda2651140171fadec5357839abe6bf05d517ce78dafc69476f024bf6b6aR325
Proof for 5.0: https://github.com/code-lts/phpmyadmin-releases/commit/5a1b8b187f2430224fa009f570304f0a4a5d965a#diff-134cdda2651140171fadec5357839abe6bf05d517ce78dafc69476f024bf6b6aR367
